src/main/webapp/view/discussion/discussion-manager.js

```diff
@@ -128,7 +128,7 @@ export default class DiscussionManager {
     for (const param in params) {
       // This is recommended by the style guide, but disallowed by linter.
       /* eslint-disable no-prototype-builtins */
-      if (params.hasOwnProperty(param)) {
+      if (params.hasOwnProperty(param) && params[param] != null) {
```

Comment:
if above is done, i think this should be changed to `!==`

Reply:
done!